test/Altinn.App.Core.Tests/Features/Maskinporten/Models/MaskinportenTokenResponseTest.cs

```diff
@@ -0,0 +1,35 @@
+using System.Text.Json;
+using Altinn.App.Core.Features.Maskinporten.Models;
+using FluentAssertions;
+
+namespace Altinn.App.Core.Tests.Features.Maskinporten.Models;
+
+public class MaskinportenTokenResponseTest
+{
+    [Fact]
+    public void ShouldDeserializeFromJsonCorrectly()
```

Comment:
Seems to me that we are simply testing System.Text.Json here? I guess one could argue that we are testing that our model is implemented correctly?

Reply:
The thought behind  this test was to see if we implement our model (and System.Text.Json markup) in a valid way as per a known response from Maskinporten. 

If they change their response without telling us, we won't catch it. If someone messes with our model in a breaking manner, the test will fail. Best I could come up with.